grafana-mixin/dashboards/grafana-overview.json

```diff
@@ -55,9 +55,7 @@
         "justifyMode": "auto",
```

Comment:
Why this changes are in this PR? Was it on purpose?

Reply:
The branch was based on the table refactor, which got merged meanwhile, so GH was comparing against master. Now it's updated with master again.